jsapp/js/components/formGallery/formGallery.actions.ts

```diff
@@ -0,0 +1,22 @@
+import type {
+  SubmissionResponse,
+  PaginatedResponse,
+} from 'js/dataInterface';
+
+export type Action =
```

Comment:
I looked if there is any recommendations and the redux recommendation is to [avoid union action types](https://redux.js.org/usage/usage-with-typescript#avoid-action-type-unions) lol.

The link's example of redux toolkit's match function makes sense. But we aren't even using redux, so it doesn't really matter. I'll rename it to avoid confusion for now.